production/loki-mixin/dashboards/loki-deletion.libsonnet

```diff
@@ -4,7 +4,8 @@ local utils = import 'mixin-utils/utils.libsonnet';
 (import 'dashboard-utils.libsonnet') {
   local compactor_matcher = if $._config.meta_monitoring.enabled
   then 'pod=~"(compactor|%s-backend.*|loki-single-binary)"' % $._config.ssd.pod_prefix_matcher
-  else if $._config.ssd.enabled then 'container="loki", pod=~"%s-backend.*"' % $._config.ssd.pod_prefix_matcher else 'container="compactor"',
+  else if $._config.ssd.enabled then 'container="loki", pod=~"%s-backend.*"' % $._config.ssd.pod_prefix_matcher
+  else 'container="compactor"',
```

Comment:
I think we could have been okay without this change? and just doing the `container="compactor"` -> `compactor_matcher` change below?

Reply:
Oh definitely, I just thought it was easier to read. If you prefer I will change this tomorrow

Reply:
I think `then 'pod=~"(compactor|%s-backend.*|loki-single-binary)"' % $._config.ssd.pod_prefix_matcher` should cover the case where we were previously used `container="compactor"` right?

Reply:
I think that's mostly right, but it looks like it wouldn't work for `%s-backend  % $._config.ssd.pod_prefix_matcher ` because AFAICT the default value for that variable is `(loki.*|enterprise-logs)`

@MichelHollands can you confirm whether we really need separate selectors for when meta monitoring is enabled?

Reply:
Yes we do unfortunately.

Reply:
What is it supposed to be? 

It looks to me like Quentins suggestion of `local compactor_matcher = 'pod=~"(compactor|%s-backend.*|loki-single-binary)"' % $._config.ssd.pod_prefix_matcher` catches all scenarios; distributed, ssd, and meta monitoring enabled.

Reply:
So what would you like me to do here?

Reply:
@MichelHollands can you confirm what the matcher should be when meta montioring is enabled? I still think @QuentinBisson suggestion of `local compactor_matcher = 'pod=~"(compactor|%s-backend.*|loki-single-binary)"' % $._config.ssd.pod_prefix_matcher` covers all cases.

Reply:
Hey @MichelHollands would you have time to check this one? :)

Reply:
@QuentinBisson lets move forward with the simplified matcher you've suggested and open follow ups with fixes if anyone reports any

Reply:
It's pushed :)